### ISSUE TITLE
random cernel panic on reboot or boot

### ISSUE THREAD

**Issue by dolphinmax** [Edoardo001/Matebook-13-Hackintosh#25]
hello. i give cernel panic in start boot. (75%) or normal boot.
i use catalina and try 1.4 1.2 etc open cores.

**Comment by Edoardo001** [comment]
Have you also windows?

**Comment by dolphinmax** [comment]
yes. win 10. (in folder EFI contains alse Windows and OS )

**Comment by Edoardo001** [comment]
Try to shutdown completely(not reboot) and then start osx

**Comment by dolphinmax** [comment]
on the contrary, when rebooting, it rarely happens when it crashes, but if 90 percent of the panic kernel

==== from log
mp_kdp_enter() timed-out on cpu 3, NMI-ing
......
timed-out during locked wait after NMI
panic(cpu 3 caller 0xffffff80112546d7 mp_rv_action().......
/library/cashes/......osfmk/i386/mp.c

**Comment by dolphinmax** [comment]
sorry for english)

**Comment by Edoardo001** [comment]
Don't know man, I haven't this problem. What version you have? Mx150 graphics? Touchscreen? 2020? 2019? I5? I7?

**Comment by dolphinmax** [comment]
graphics no. (only i5 8265U kaby lake and Intel UHD Graphics 620) no tachscreen (year 2018 WX9)

**Comment by dolphinmax** [comment]
and i change ssd HP EX900  500GB

**Comment by Edoardo001** [comment]
Matebook 13 2018??? Are you sure you have matebook 13?

No one in the group have this problem. Take some photos of your bios settings

**Comment by dolphinmax** [comment]
![IMG_0190](https://user-images.githubusercontent.com/8274797/85790224-93909880-b738-11ea-8176-cf88601740d7.jpg)

**Comment by dolphinmax** [comment]
![IMG_0191](https://user-images.githubusercontent.com/8274797/85790629-2f220900-b739-11ea-81d1-33d60142b8b0.jpg)

**Comment by Edoardo001** [comment]
Disable tcm/tpm don't know if can cause that panic. Try a fresh install if persist or check the panic reason

**Comment by dolphinmax** [comment]
thanks .. I will try disable tcm/tpm and write.  before and on clean I already reinstalled catalina /no result .

**Comment by dolphinmax** [comment]
it still freeze at startup. Three times booted to the 4th hang. I noticed maybe I went into the network settings before shutting down. and problem boot. if i enable -v , bool very slow ,strings scroll very slow 1 for 10sec )

**Comment by dolphinmax** [comment]
model on case wrt-w19 (2019) 
![IMG_0197](https://user-images.githubusercontent.com/8274797/85833093-da62aa80-b799-11ea-9602-4ab4daea0e16.jpg)

**Comment by Edoardo001** [comment]
Try disabling in config.plist, Gpu disabler ssdt.

**Comment by dolphinmax** [comment]
and installed windows can not affect when loading?

**Comment by dolphinmax** [comment]
> Try disabling in config.plist, Gpu disabler ssdt.

no ( not worked

**Comment by Edoardo001** [comment]
Don't know man

**Comment by dolphinmax** [comment]
It seems on the opencoreinstallation folder it works stably. What is the difference between it and the opencorepost?

**Comment by Edoardo001** [comment]
> It seems on the opencoreinstallation folder it works stably. What is the difference between it and the opencorepost?

Opencore installation contain the things you need for the installation, not all the ssdt and parches

**Comment by Edoardo001** [comment]
> It seems on the opencoreinstallation folder it works stably. What is the difference between it and the opencorepost?

Maybe i've experienced same problem, the laptop stuck at 1/10 of the loading bar. Have you same behaviour? are you using itlwm kext?

**Comment by dolphinmax** [comment]
Yes. he always got stuck at the start of the loading scale. I am now with Opencoreinstall and there was no more problem (10-15 tests).

**Comment by dolphinmax** [comment]
Opencoreinstall (by default no changes)

**Comment by Edoardo001** [comment]
> Yes. he always got stuck at the start of the loading scale. I am now with Opencoreinstall and there was no more problem (10-15 tests).

Can you try post install without ssdt usb(disabling in config.plist?

**Comment by Edoardo001** [comment]
Seems  fixed in latest version, can you try and report back?

**Comment by dolphinmax** [comment]
Hello! Thenx for new version. I  i keep testing OpenCorePost v1.5 for Intel HD graphics only version (3-4 loads) its OK. no frezez no panic. I’ll test and write tomorrow.

one feature. Sometimes it loads quickly, changing the resolution in the middle of the load (apple less). And now, without changing the resolution longer. But it booted and shutdown slow  and next boot fast and shutdown fast.

**Comment by dolphinmax** [comment]
upd. no panic . worked!

**Comment by Edoardo001** [comment]
> Hello! Thenx for new version. I  i keep testing OpenCorePost v1.5 for Intel HD graphics only version (3-4 loads) its OK. no frezez no panic. I’ll test and write tomorrow.
> 
> one feature. Sometimes it loads quickly, changing the resolution in the middle of the load (apple less). And now, without changing the resolution longer. But it booted and shutdown slow  and next boot fast and shutdown fast.

Can you add "-v" in boot arguments to boot verbose and check what change between the speed and the slow boot?
